web/Views/Admin/PropertyList.cshtml

```diff
@@ -1,113 +1,56 @@
 ﻿@model PropertiesDashboardViewModel
 
 @{
-    ViewData["Title"] = "All Properties List";
+    ViewData["Title"] = "Properties";
 }
 
-<h2>All Properties List</h2>
+<div class="mx-md-5">
+    <h1 class="font-h1 mt-5">Properties</h1>
 
-<div class="mb-3">
-    <div class="row">
-        <div class="col-4">
-            @using (Html.BeginForm("SortProperties", "Property", FormMethod.Get))
-            {
-                <select class="form-select" id="sortBy" name="sortBy">
+    <div class="row mt-4">
+        <div class="col-auto me-auto mb-2">
+            <form asp-action="SortProperties" asp-controller="Property" method="get" class="row mx-0">
+                <select class="form-select col h-auto me-3" id="sortBy" name="sortBy">

```

Comment:
This won't stay filled when we send this form and get back to this page, could we possibly make this a DropdownListFor like in the Landlord list :)